language/stdlib/modules/ValidatorConfig.move

```diff
@@ -231,6 +231,17 @@ module ValidatorConfig {
         ensures result == spec_get_operator(addr);
     }
 
+    /// Return the human-readable name for the validator account.
+    /// Aborts if `addr` does not have a `ValidatorConfig` resource.
+    public fun human_name(addr: address): vector<u8> acquires ValidatorConfig {
```

Comment:
May be we can call it `get_human_name` to be consistent with the rest of the getter functions?